tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "rootDir": "src",
     "strict": true,
     "target": "es2020",
-    "lib": ["es2020"]
+    "lib": ["dom", "es2020"]
```

Comment:
This would allow referencing DOM types in the project. We may want to add a comment here to follow up and remove.

Reply:
I added a comment and will get it pushed up in my next PR